sdk/eventhubs/azure-messaging-eventhubs-track1-perf/src/main/java/com/azure/messaging/eventhubs/perf/EventProcessorOptions.java

```diff
@@ -0,0 +1,51 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+package com.azure.messaging.eventhubs.perf;
+
+import com.beust.jcommander.Parameter;
+
+/**
+ * Options for Event Processor tests.
+ *
+ * @see EventProcessorTest
+ */
+public class EventProcessorOptions extends EventHubsReceiveOptions {
+    @Parameter(names = {"-scs", "--storageConnectionString"}, description = "Connection string for Storage account.",
+        required = true)
+    private String storageConnectionString;
+
+    @Parameter(names = {"-se", "--storageEndpoint"}, description = "Endpoint for storage account",
+        required = true)
+    private String storageEndpoint;
+
+    @Parameter(names = {"-e", "--eventsToSend"}, description = "Number of events to send per partition.")
+    private int eventsToSend = 100000;
```

Comment:
May add event payload size as a parameter

Reply:
Base class PerfStressOptions expose a .getSize() (-s, --size) parameter.

Reply:
We use this in ServiceTest line 62.